src/General_Spotify_Helpers.py

```diff
@@ -387,6 +392,8 @@ def get_playlist_tracks(self, playlist_id: str,
     """""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
     def create_playlist(self, name: str, description: str='', public: bool=False) -> str:
         self._validate_scope(["playlist-modify-public", "playlist-modify-private"])
+        if len(self.get_user_playlists()) > 400:
```

Comment:
Spoopy number

Reply:
it's not spoopy it's M A G I C.
Don't care at the moment to abstract to a constant. Almost all magic numbers, strings, filepaths will be entered into a 'settings' type file under BRO-86.